modmail/utils/embeds.py

```diff
@@ -61,7 +61,7 @@ def __init__(self: discord.Embed, description: str = None, **kwargs):  # noqa: N
         self.set_author(
             name=author_name if author_name is not None else author.name,
```

Comment:
```suggestion
            name=author_name or author.name,
```

Reply:
This was added in a different pr and changing it is out of scope of this pr.